### ISSUE TITLE
testConnection() always returns FALSE

### ISSUE THREAD

**Issue by jbrepogmailcom** [shurillu/CTBot#12]
Hello! Is your basic (LED) example working? About two weeks ago it worked for me fine, but now it does not connect to Telegram server and testConnection() always returns FALSE. I tried to get new token, but still does not connect. Connect to WiFi OK, I have added 
 Serial.print("Connecting to wifi... ");
 Serial.println(myBot.wifiConnect(ssid, pass));
and it returns 1.

Thank you

**Comment by shurillu** [comment]
Hello,

   just few questions:

1) Have you updated the Arduino Libraries? Remember that the ArduinoJson version Library MUST be 5.13.2 (or lesser... anyway is safer using THAT version).

2) What version of CTBot are you using? The one downloaded form Arduino IDE or the master one (directly downloaded form GitHub).

3) Have you double checked the Telegram Bot Token?

I have just tested the "lightbot" example (with my wifi credentials and the correct Telegram Token) and simply work.

Please, let me know if the issue still exist, as I can help you and edventually find the bug.

Regards, Stefano

P.S.: I'm using a NodeMCU 1.0

**Comment by jbrepogmailcom** [comment]
Thanks for the response. Yes all libraries are latest version as of Dec-28, except ArduinoJson. That is as instructed 5.13.2 (I also tried 5.13.4, no change). Version of CBot 1.2.0. I do not know what version was current at beginning of December, but Telegram worked and I could connect to server. Token is OK, I tried to renew it, no change. I will try again in a few days. Now I use Plan B - Blynk.
